bindings/r/LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2024
+COPYRIGHT HOLDER: treesitterr authors
```

Comment:
Not sure if this is ever destined for CRAN, but I vaguely recall the CRAN maintainers requesting the copyright statement include the author names explicitly.